brokerapi/brokers/account_managers/sql_account_manager.go

```diff
@@ -115,25 +127,27 @@ func (sam *SqlAccountManager) DeleteAccountFromGoogle(binding models.ServiceBind
 		return fmt.Errorf("Error creating CloudSQL client: %s", err)
 	}
 
-	op, err := sqlService.SslCerts.Delete(sam.ProjectId, instance.Name, sqlCreds.Sha1Fingerprint).Do()
-	if err != nil {
-		return fmt.Errorf("Error deleting ssl cert: %s", err)
-	}
+	if sqlCreds.CaCert != "" {
+		certOp, err := sqlService.SslCerts.Delete(sam.ProjectId, instance.Name, sqlCreds.Sha1Fingerprint).Do()
```

Comment:
any risk this times out?

Reply:
It completes quickly and then we have to wait for the operation to complete below. So I don't think it would timeout.